vcpkg/users/platforms/xbox.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Xbox
+description: Use x64 community triplets for Xbox development.
+ms.date: 04/27/2023
```

Comment:
```suggestion
title: Xbox
description: Use x64 community triplets for Xbox development.
author: walbourn
ms.author: viromer
ms.date: 8/8/2023
```

Feel free to change `ms.author` to your MS alias if you want to be notified about changes to this file.